RomanHumeniuk/hwTable/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Большая двадцатка G20</title>
+  </head>
+  <body>
+    <table border='2' align='center' width='50%'>
+      <tr align="center" bgcolor='lightgreen'>
+        <td colspan="2">
+          <h1>G20 Большая двадцатка</h1>
+        </td>
+      </tr>
+      <tr>
+        <td bgcolor='lightgrey'>
+          <ul>
+            <li><a href='about.html'>Создание G20</a></li>
+            <li><a href='article.html'>История G20</a></li>
+            <li><a href='home.html'>Страны участники</a></li>
+          </ul>
+        </td>
+        <td bgcolor="darkgrey">
+          <center><h2>G20</h2></center>
```

Comment:
center is inline tag, h2 is block tag so
center should be under h2

< h2> < center> some text </ center> </ h2>